File: app/src/main/java/buu/example/myapp/BOOK.java
package buu.example.myapp;

public class BOOK {
 private String BOOKID;
    private String ANTHOR;
    private String PAGE;

    public BOOK() {
    }

    public BOOK(String BOOKID, String ANTHOR, String PAGE) {
        this.BOOKID = BOOKID;
        this.ANTHOR = ANTHOR;
        this.PAGE = PAGE;
    }

    public String getBOOKID() {
        return BOOKID;
    }

    public void setBOOKID(String BOOKID) {
        this.BOOKID = BOOKID;
    }

    public String getANTHOR() {
        return ANTHOR;
    }

    public void setANTHOR(String ANTHOR) {
        this.ANTHOR = ANTHOR;
    }

    public String getPAGE() {
        return PAGE;
    }

    public void setPAGE(String PAGE) {
        this.PAGE = PAGE;
    }
}

File: app/src/main/java/buu/example/myapp/database/Book.kt
package buu.example.myapp.database



import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "book")
class Book(
    @PrimaryKey(autoGenerate = true)
    var bookId: Int =0 ,

    @ColumnInfo(name = "bookname")
    var bookname : String?,

    @ColumnInfo(name = "anthor")
    var anthor : String?,

    @ColumnInfo(name = "page")
    var page : String?




)